metrics/grafana/tikv_details.json

```diff
@@ -19396,7 +19396,7 @@
               "hide": false,
               "interval": "",
               "intervalFactor": 2,
-              "legendFormat": "pending-bytes-{{instance}}",
+              "legendFormat": "pending-compaction-bytes-{{instance}}",
```

Comment:
As a legend, I think "pending-bytes" should be clear enough because it's under the "Pending compaction bytes" panel. A too long legend may squeeze the space of the graph too much.

Reply:
Thank you for your comments.
I agree with your thought. I will revert legendFormat.